provisioner/template.go

```diff
@@ -184,6 +185,9 @@ func renderTemplate(context *templateContext, filePath string, data interface{})
 		"base64":                    base64Encode,
 		"manifestHash":              func(template string) (string, error) { return manifestHash(context, filePath, template, data) },
 		"autoscalingBufferSettings": autoscalingBufferSettings,
+		"asgSize":                   asgSize,
+		"azId":                      azId,
```

Comment:
nit: I would call this function `azID` (https://github.com/golang/go/wiki/CodeReviewComments#initialisms).